catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/CollectionDAO.java

```diff
@@ -779,12 +780,38 @@ List<List<String>> listFromByAllPrefix(
             + "fromFQN LIKE CONCAT(:fqnPrefix, '%') AND fromType = :fromType AND toType = :toType "
             + "AND relation = :relation")
     @RegisterRowMapper(ToFieldMapper.class)
-    List<List<String>> listToByPrefix(
+    List<Triple<String, String, String>> listToByPrefix(
         @Bind("fqnPrefix") String fqnPrefix,
         @Bind("fromType") String fromType,
         @Bind("toType") String toType,
         @Bind("relation") int relation);
 
+    @SqlQuery(
+        "SELECT fromFQN, toFQN, json FROM field_relationship WHERE "
+            + "fromFQN = :fqn AND fromType = :type AND toType = :otherType AND relation = :relation "
+            + "UNION "
+            + "SELECT toFQN, fromFQN, json FROM field_relationship WHERE "
+            + "toFQN = :fqn AND toType = :type AND fromType = :otherType AND relation = :relation")
+    @RegisterRowMapper(ToFieldMapper.class)
+    List<Triple<String, String, String>> listBidirectional(
+        @Bind("fqn") String fqn,
```

Comment:
We have EntityRepository.findBoth() method that can be replaced with this method call.

Reply:
I think` EntityRepository.findBoth()` is for looking up EntityRelationships, not FieldRelationships. I can add similar querries for EntityRelationships, but I think that should go in another PR